core/messaging/__init__.py

```diff
@@ -1,13 +1,19 @@
+import math
+import re
 from abc import ABC, abstractmethod
 
 
 class Message:
     """Generic Message class that defines the components of a validation error message"""
 
+    cell_index_pattern = re.compile(r"^([a-zA-Z]+)(\d+)?$")
```

Comment:
Do we need to allow lowercase letters?

Reply:
Can't tell if this was before or after I allowed lowercase - do you think we should, or shouldn't?

Reply:
I was going to suggest we shouldn't, assuming this doesn't break anything existing

Reply:
don't think it breaks anything - I'll remove it